.gitignore

```diff
@@ -1,8 +1,10 @@
 .DS_Store
-vendor
-build
+/vendor
+/build
 node_modules
 wiki
 amp.zip
 **/*-compiled.js
 built
+/amphtml
+.vscode
```

Comment:
This belongs to one's personal global `.gitignore` file. Same with `.DS_Store`, actually.

Reply:
We don't want these every to be committed, so I'd rather include them here than rely on a user to populate their global `.gitignore` (which I doubt few do).

Reply:
How is that change related to this PR then?

It‘s a user-specific folder that the user is responsible for excluding. If they don’t, we would catch it in the PR anway. Otherwise we‘d have to add entries for all the dozens or so popular editors out there, which is not sustainable. Or is the VSC folder in particular accidentally part of lots of PRs so far?

Reply:
It's related to this PR because I added `/amphtml` to the gitignore, and I noticed that `.vscode` was also among the uncommitted files. I'll just remove it.